### ISSUE TITLE
Import and require not working

### ISSUE THREAD

**Issue by Apsysikal** [meodai/fettepalette#17]
I installed it via npm and wether i import or require the module I always get errors.
I tried it via the „Try on runkit“ button on npm. It fails there aswell. Is there something I‘m missing?

**Comment by Apsysikal** [comment]
I've updated the package.json in the node_modules folder to contain:

```
...,
"exports": {
    "require": "./dist/index.cjs",
    "import": "./dist/index.mjs"
},
...
```

It is now working but it seems to me that changing a file in the node_modules folder probably should not be the fix for everyone.

**Comment by meodai** [comment]
hey @Apsysikal we have an issue for that: https://github.com/meodai/fettepalette/issues/13
I'm a bit confused how to address that. Feel free to create a pull-request if you have any idea

**Comment by meodai** [comment]
so I did that, but 

```javascript
import {
  generateRandomColorRamp
} from 'fettepalette';

console.log(generateRandomColorRamp) // => undefined
```

at least with parcel.js

**Comment by Apsysikal** [comment]
I don't work with parcel. So i can't give you any input on that. It fixed it for my use case, which is generating some hex codes from the command line with a little js file. You could leave this issue open for other people with the same problem.

**Comment by meodai** [comment]
Yes please! Does it work for you now though? (the current version)
